### ISSUE TITLE
04/09/23_NY_Android_After entering a mobile number, talkback reading should be a single single digit_1980

### ISSUE THREAD

**Issue by AlluGowthami28** [nammayatri/nammayatri#2885]
#1980 
After entering a mobile number, talkback reading should be a single single digit .
**Steps to Reproduce**
1. Launch the application
2. Tap on get started 
3. Enter mobile number and observe the screen

**Actal Result**
After entering mobile number, talkback not reading a single single digit .

**Expected Result** 
After entering a mobile number, talkback reading should be a single single digit .

**Comment by Vinaykm999** [comment]
@MercyQueen This issue is still exist, marking it as open state.

**Comment by Vinaykm999** [comment]
[Check this screen record](https://drive.google.com/file/d/1NNoTpIXeQ1VQGQffII4rw90lJuU_73_Q/view?usp=sharing)
